study.md

```diff
@@ -38,29 +38,74 @@ As it makes sense to you please write the following in JSON format:
 -   Has lived in two places recently: 123 Fake street, and 2 Muchinfo Road
 
 ```json
-<!-- your answer here -->
+{'Jason': {
+    'pet': {
+      'Mr. Kitty'
+    },
+    'Roommates': {
+      'Dave',
+      'Miller',
+    },
+    'Sports': {
+      'Red Sox',
+      'Patriots',
+      'Bruins',
+    },
+    'Addresses': {
+      '123 Fake street',
+      '2 Muchinfo Road',
+    }
+  }}
 ```
 
 I forgot, my roomate Dave has two goldfish, one named Bob (he's red) and the
 other named Mr.MagicNibbles (yellow) and bites a bit. Please copy the JSON you
 already wrote and include the information about Dave's fish.
 
 ```json
-<!-- your answer here -->
+{'Jason': {
+    'pet': {
+      'Mr. Kitty'
+    },
+    'Roommates': {
+      'Dave': {
+        'pet': {
+          'goldfish': {
+            'Bob': {
+              'red',
+            }, 'Mr. MagicNibbles': {
+              'yellow',
+              'bites a bit',
+            }
+          }
+        }
+      },
+      'Miller',
+    },
+    'Sports': {
+      'Red Sox',
+      'Patriots',
+      'Bruins',
+    },
+    'Addresses': {
+      '123 Fake street',
+      '2 Muchinfo Road',
+    }
+  }}
 ```
 
 ## JSON Methods
 
 Using the JSON your wrote above, can you write a method on the JSON?
 
 ```json
-<!-- your answer here -->
+I don't understand the question being asked. Am I to write a javascript method using the JSON above?
 ```
 
```

Comment:
These questions are asking you if you can write a method or comment on JSON and the answer to both is "No JSON is a string"